docs/setup/production.asciidoc

```diff
@@ -109,6 +109,35 @@ transport.tcp.port: 9300 - 9400
 elasticsearch.hosts: ["http://localhost:9200"]
 --------
 
+=== Load Balancing Across Multiple Kibana Instances
+Multiple Kibana installations can be served behind a load balancer with configuration changes.  Descriptions can be referenced on the {kibana-ref}/settings.html[settings page].
```

Comment:
Is there a user action in the first sentence? Would this work?

To serve multiple Kibana installations behind a load balancer, you must change the configuration.  See  {kibana-ref}/settings.html[Configuring Kibana] for details on each setting.

Reply:
:thu! de4f19b